packages/hyperion-autologging/src/ALElementValuePublisher.ts

```diff
@@ -132,12 +136,12 @@ export function publish(options: InitOptions): void {
              * We know we are lookging for checkbox/radio inputs, so checked is good enough
              * Also, the following check ensures we are not reading PII from other inputs, in case we expand the search
              */
-            if (input.checked) {
+            if (includeInitialDisabledState || input.checked) {
               emitEvent({
                 surface,
                 element,
                 relatedEventIndex,
-                value: 'true',
+                value: String(input.checked),
```

Comment:
Can we change the value type so that it's a boolean and not a stringified boolean?